index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+      <h1 id="title">Tribute Page</h1>
+      <div id="img-div">
+        <img
+          id="image"
+          src="https://pbs.twimg.com/media/DGVBXfVV0AEyqXH.jpg"
+          alt="tribute-image"
+        />
+        <p id="img-caption">This is just an image</p>
```

Comment:
Great job so far describing your picture, but as we should follow the semantic HTML, I recommend you changing this <p> tag to be <figcaption> for the SEO sake. I strongly encourage you to read this [article](https://www.w3schools.com/html/html5_semantic_elements.asp) about how to perfectly describe your HTML tags. 😉